allennlp/semparse/__init__.py

```diff
@@ -29,8 +29,7 @@
 # dependency issues.  If you want to import semparse stuff from the data code, just use a more
 # complete path, like `from allennlp.semparse.worlds import WikiTablesWorld`.
 from allennlp.data.tokenizers import Token as _
-from allennlp.semparse.domain_languages.domain_language import (DomainLanguage, ParsingError,
-                                                                ExecutionError,
-                                                                predicate, predicate_with_side_args)
+from allennlp.semparse.common.errors import ParsingError, ExecutionError
+from allennlp.semparse.domain_languages.domain_language import (DomainLanguage, predicate, predicate_with_side_args)
```

Comment:
I think this is the line that pylint is complaining is too long; if you remove the parentheses, it might just barely fit.

Reply:
Should we fixed now.